integ-tests/src/main/java/com/google/base/TestBase.java

```diff
@@ -1,4 +1,49 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package com.google.base;
 
-public class TestBase {
-}
\ No newline at end of file
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/** Base class with general values for all Junit test suites */
+public abstract class TestBase {
+
+  public static final String DB_NAME = ""; // TODO get from a parameter
+  public static final String USERNAME_DB = ""; // TODO get from a parameter
+  public static final String PASSWORD_DB = ""; // TODO get from a parameter
+  public static final String URL_DB = "jdbc:teradata://localhost/DBS_PORT=1025";
+  public static final String ET_OUTPUT_PATH = ""; // TODO get from a parameter
+
+  /**
+   * @param dateString Date passed as String.
+   * @return Date as a timestamp as Long.
+   */
+  public static long getTimestampByDate(String dateString) {
+    if (dateString.equals("")) {
```

Comment:
Should this maybe rather throw an `IllegalStateException`, too. Maybe also on `null`?

Reply:
I can fix it a bit differently, I can call `getStringNotNull()` and it securely passes either a string or an empty string.
```
getTimestampByDate(JdbcHelper.getStringNotNull(rs, "LastAccessTimeStamp"));
```